src/main/java/reposense/util/FileUtil.java

```diff
@@ -110,6 +111,26 @@ public static void zipFoldersAndFiles(List<Path> pathsToZip,
         }
     }
 
+    /**
+     * Writes the ignore revs file containing the {@code ignoreCommitList} at the given {@code path}.
+     * @return An Optional containing the Path to the ignore revs file, or an empty Optional
+     *         if there was an error while writing the ignore revs file.
+     */
+    public static Optional<Path> writeIgnoreRevsFile(String path, List<CommitHash> ignoreCommitList) {
+        String contentOfIgnoreRevsFile = ignoreCommitList
+                                            .stream()
+                                            .map(CommitHash::toString)
+                                            .reduce("", (hashes, newHash) -> hashes + newHash + "\n");
+
+        try (PrintWriter out = new PrintWriter(path)) {
```

Comment:
I don't think use this type of try-catch statement in this project. Can the expression be placed within the try block?

Reply:
It's actually used right above it in another function
![Screenshot 2021-09-05 at 11 47 22 PM](https://user-images.githubusercontent.com/61911161/132132995-55ce97e7-9c38-491b-a094-a77e150d6447.png)

Reply:
Ok! Then let's keep it this way.